presto-cli-rpm/pom.xml

```diff
@@ -0,0 +1,165 @@
+                            <release>1</release>
+
+                            <group>Applications/Databases</group>
+                            <description>Presto CLI RPM Package.</description>
+                            <architecture>noarch</architecture>
+
+                            <links>
+                                <link>
+                                    <path>/usr/local/bin/presto-cli</path>
+                                    <target>/usr/lib/presto-cli/presto-cli-${project.version}-executable.jar</target>
+                                </link>
+                            </links>
+
+                            <rules>
+                                <rule>
+                                    <destination>/usr/lib/presto-cli</destination>
+                                    <base>${cli.tar.package}</base>
+                                    <!-- make sure jar is executable -->
+                                    <fileMode>0755</fileMode>
+                                    <includes>
+                                        <include>presto-cli-${project.version}-executable.jar</include>
+                                    </includes>
+                                </rule>
+                            </rules>
+                        </package>
+                    </packages>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>enforce-file-size</id>
+                        <phase>verify</phase>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <requireFilesSize>
+                                    <!-- CLI RPM shouldn't be too large -->
+                                    <maxsize>31457280</maxsize>
+                                    <files>
+                                        <file>${project.build.directory}/${project.build.finalName}.noarch.rpm</file>
+                                    </files>
+                                </requireFilesSize>
+                            </rules>
+                            <fail>true</fail>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+    <profiles>
+        <profile>
+            <id>ci</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <version>2.22.2</version>
```

Comment:
Please extract `maven-failsafe-plugin` version to `pom.xml` and use that version here and  `presto-server-rpm`

Reply:
Done.